exercises/concept/luigis-luscious-lasagna/.docs/introduction.md

```diff
@@ -0,0 +1,85 @@
+# Introduction
```

Comment:
It looks like the contents of this file match the concept's `about.md`, but it should be much closer to the `introduction.md`. As per [the docs](https://exercism.org/docs/building/tracks/concept-exercises#h-file-docs-introduction-md), the exercise's `introduction.md` file should give the student just enough context to figure out the solution themselves.